pages/k8s/1.25/components.md

```diff
@@ -0,0 +1 @@
+*** AUTOGENERATED *** (I hope)
```

Comment:
When there is something we can point the generator script at, we can create this